theSrc/scripts/LabeledScatter.js

```diff
@@ -318,9 +320,17 @@ class LabeledScatter {
     return nsewdrag[0][0].getBBox()
   }
 
-  legendRect () {
+  plotlyLegendRect () {
     const legend_bg = d3.select(this.rootElement).select('.legend rect.bg')
-    return legend_bg[0][0].getBBox()
+    const result = legend_bg[0][0].getBBox()
+    const ctm = legend_bg[0][0].getCTM()
+    result.x = ctm.e
+    result.y = ctm.f
+    return result
+  }
+
+  isPlotlyLegendOnRight (plotly_chart_layout) {
+    return plotly_chart_layout.legend.orientation === 'v' && plotly_chart_layout.legend.x >= 1
```

Comment:
Was this also from my code in flipStandardCharts? It might not be quite right. Sometimes people put the legend just inside the chart area because they are trying to save space. Is it possible to compare the right coord of the legend against the right coord of nsewdrag_rect instead?

Reply:
Hmm OK, that might be better